keps/sig-network/4559-kubelet-probes/README.md

```diff
@@ -169,42 +139,34 @@ See further discussion in [kubernetes #102613].
 [kubernetes #101324]: https://github.com/kubernetes/kubernetes/issues/101324
 [the NetworkPolicy has not yet been fully programmed]: https://kubernetes.io/docs/concepts/services-networking/network-policies/#pod-lifecycle
 [readiness gates]: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-readiness-gate
-[Exec probes are much, much slower than network probes]: https://medium.com/netcracker/exec-probes-a-story-about-experiment-and-relevant-problems-12de616c0a76
-[kubernetes #89898]: https://github.com/kubernetes/kubernetes/issues/89898
-[kubernetes #115143]: https://github.com/kubernetes/kubernetes/pull/115143
 [kubernetes #102613]: https://github.com/kubernetes/kubernetes/issues/102613
-[kubevirt]: https://github.com/kubevirt/kubevirt/blob/main/docs/devel/networking.md#vmi-networking
 
 ## Motivation
 
 ### Goals
 
 - Define new probe types and deprecate existing probe types.
 
-    - The alternative is to change the semantics of existing probes but that is
-      harder to achieve without also changing the user-facing API semantics.
+    - The proposals below have semantics that are just different enough from
+      the current probes that there would be compatibility problems in some
+      cases if we just changed existing probes to use the new semantics.
 
     - Continue to backward-compatibly support all existing probe handlers
-      and lifecycle handlers by default (at least for a while). Older probe
-      types can be blocked via Pod Security admission, so that administrators
-      can create NetworkPolicies that don't include the kubelet probe hole.
+      and lifecycle handlers. Administrators who want to create
+      AdminNetworkPolicies that would block traditional kubelet probes
+      can use a ValidatingAdmissionPolicy to prevent users from creating
```

Comment:
Discussion ruled out the possibility of using PSA to deprecate things that aren't insecure, so they'd have to use VAP or something.